crowbar_framework/vendor/assets/javascripts/jquery/spritely.js

```diff
@@ -1,539 +0,0 @@
-/*
```

Comment:
> Can you keep this file just for now? Just to be safe (in case it's used in some hidden corner we forgot about).

I am absolutely sure that this is not used ANYWHERE else!